pfcon/pfcon.py

```diff
@@ -434,29 +448,50 @@ def dataRequest_process(self, *args, **kwargs):
             if k == 'return':           d_return            = v
             if k == 'key':              str_key             = v
             if k == 'op':               str_op              = v
-
-
         d_meta                  = d_request[str_metaHeader]
 
         if str_op == 'pushPath':
             d_pushPath = self.dataRequest_processPushPath(d_meta = d_meta)
 
+        # pudb.set_trace()
         str_remoteService       = d_meta['service']
         str_dataServiceAddr     = Gd_tree.cat('/service/%s/data/addr'       % str_remoteService)
         str_dataServiceURL      = Gd_tree.cat('/service/%s/data/baseURLpath'% str_remoteService)
-
-        dataComs = pfurl.Pfurl(
-            msg                         = json.dumps(d_request),
-            verb                        = 'POST',
-            http                        = '%s/%s' % (str_dataServiceAddr, str_dataServiceURL),
-            b_quiet                     = False,
-            b_raw                       = True,
-            b_httpResponseBodyParse     = True,
-            jsonwrapper                 = ''
-        )
-
+        b_enableTokenAuth       = Gd_tree.cat('/service/%s/data/enableTokenAuth'% str_remoteService)
```

Comment:
Do we still need the enable auth option at all now?  Can we not just say it's always enabled and the default token is passed unless you change it?